docs/stencil-docs/storefront-customization/custom-templates.md

```diff
@@ -118,7 +118,9 @@ Beyond the single URL mapped to each template in the above examples, you have th
 
 ## Specifying custom front matter
 
-You can't specify front matter on a custom template, but you can still access front matter-injected data. Custom templates inherit front matter from their corresponding default template. Specify the data you would like your custom template to access in the default template's front matter. See [Using Front Matter](/stencil-docs/storefront-customization/using-front-matter) for more information on using front matter. The following example shows data you can inject into `templates/pages/brand.html` to access data in `templates/pages/custom/brand/custom-brand.html`. After [mapping](/stencil-docs/storefront-customization/custom-templates#mapping-multiple-urls) the custom template to the default page, the cart object displays on the page.
+You can't specify front matter on a custom template, but you can still access front matter-injected data. Custom templates inherit front matter from their corresponding default template. Specify the data you would like your custom template to access in the default template's front matter. See [Using Front Matter](/stencil-docs/storefront-customization/using-front-matter) for more information on using front matter. 
+
+The following example shows data you can inject into `templates/pages/brand.html` to access data in `templates/pages/custom/brand/custom-brand.html`. After [mapping](/stencil-docs/storefront-customization/custom-templates#mapping-multiple-urls) the custom template to the default page, the cart object displays on the page.
```

Comment:
@bc-traciporter I believe we can remove lines 121 -123 above. Stencil-cli versions `5.2.1+` supports front matter on custom templates. So if a customer defines a custom template with a front matter in his local theme (in `config.stencil.json`), this front matter will be taken into account by stencil-cli. So you don't need to change default template/front matter as we use directly what we have in custom template.